python_modules/libraries/dagster-aws/dagster_aws/s3/sensor.py

```diff
@@ -65,6 +65,7 @@ def get_objects(
         for idx, obj in enumerate(sorted_objects):
             if obj.get("LastModified") > since_last_modified:
                 return sorted_objects[idx:]
+        return []
```

Comment:
Could we add a comment to describe what is happening and a test in `dagster_was_test/s3_tests/test_s3_sensor.py`?